### ISSUE TITLE
Double validation, while you have missed answering the question

### ISSUE THREAD

**Issue by Quer1234** [DevAdventCalendar/DevAdventCalendar#436]
1. Create at least two questions
2. As an admin close first one (user didn't answer for a question) and leave next one open
3. Re-log as a user and try to click for a question, for which user didn't answered and is already closed

Actual results
There are several results after that situation.
1. User is redirected to the below screen (It is OK)
![image](https://user-images.githubusercontent.com/57844124/99433927-77231180-290e-11eb-99e1-469cb7fdf226.png)

2. There is visible message at the screen 'Niestety, spóźniłeś się....'
![image](https://user-images.githubusercontent.com/57844124/99433866-62df1480-290e-11eb-9d87-55045d266a53.png)

3. User click at the question. For a few second message 'Niestety, spóźniłeś się....' is visible, and after thar user is redirected to the proper screen, with information 'Niestety, spóźniłeś się z odpowiedzią na to pytanie :('

**Comment by Quer1234** [comment]
If user will click not on 'logo partnera dnia' information 'niestety spozniles sie' will be visible
If user will click on 'logo partnera dnia' will be redirected to the other screen

**Comment by WTobor** [comment]
Good point. It is a low priority because we will close the question at the end of the competition period.
